src/Html.php

```diff
@@ -804,17 +761,21 @@ private static function booleanInput($type, $name, $checked = false, $options =
      *   except that the array keys represent the optgroup labels specified in $items.
      * - encodeSpaces: bool, whether to encode spaces in option prompt and option value with `&nbsp;` character.
      *   Defaults to false.
-     * - encode: bool, whether to encode option prompt and option value characters.
-     *   Defaults to `true`.
+     * - encode: bool, whether to encode option prompt and option value characters. Defaults to `true`. This option is
+     *   available.
      *
-     * The rest of the options will be rendered as the attributes of the resulting tag. The values will
-     * be HTML-encoded using {@see encode()}. If a value is null, the corresponding attribute will not be rendered.
+     * The rest of the options will be rendered as the attributes of the resulting tag. The values will be HTML-encoded
+     * using {@see encode()}. If a value is null, the corresponding attribute will not be rendered.
      * See {@see renderTagAttributes()} for details on how attributes are being rendered.
      *
      * @return string the generated drop-down list tag
      */
-    public static function dropDownList($name, $selection = null, $items = [], $options = []): string
-    {
+    public static function dropDownList(
```

Comment:
It was OK before.

Reply:
PSR-2 max 120 characters for line, like the new PSR-12.

Reply:
There is no hard limit.